data/json/ammo_effects.json

```diff
@@ -115,18 +120,88 @@
   {
     "id": "EXPLOSIVE_RAUFOSS",
     "type": "ammo_effect",
-    "explosion": {
-      "power": 2.4,
-      "distance_factor": 0.6,
-      "fire": true,
-      "shrapnel": { "casing_mass": 28, "fragment_mass": 1.4, "recovery": 0, "drop": "null" }
-    }
+    "explosion": { "power": 2.4, "shrapnel": { "casing_mass": 28, "fragment_mass": 14 } }
   },
   {
     "id": "FRAG",
     "type": "ammo_effect",
     "explosion": { "power": 185, "shrapnel": { "casing_mass": 212, "fragment_mass": 0.05 } }
   },
+  {
+    "id": "20x66_FRAG",
```

Comment:
I would prefer using EXPLOSION/FRAG prefix instead of suffix.

Reply:
Changed all the ammo effects to use prefixes